VideoWeb/VideoWeb/ClientApp/src/app/shared/chat-body-window/chat-body-window.component.scss

```diff
@@ -1,44 +1,94 @@
-@import '~govuk-frontend/govuk/all';
+@import '~govuk-frontend/govuk/base';
 
-.message-container {
-  width: inherit;
-  margin: 0px 2px !important;
+.failed-message {
+  @include govuk-font($size: 16, $weight: 'bold');
+  display: block;
+  margin-bottom: govuk-spacing(0);
+  clear: both;
+  color: $govuk-error-colour;
 }
 
-.own-message {
-  text-align: left;
+.alert-image {
+  width: 14px;
+  height: 14px;
 }
 
-.other-message {
-  text-align: right;
+.spacer {
+  height: 10px;
+  padding: 0px !important;
+  float: left;
+}
+
+.messages-container {
+  font-family: 'GDS Transport', arial, sans-serif;
```

Comment:
can we not use `@include govuk-font` here?

Reply:
👍